pages/common/mongoexport.md

```diff
@@ -0,0 +1,32 @@
+# mongoexport
+
+> Produce exports of data stored in a MongoDB instance formatted as JSON or CSV.
+> More information: <https://docs.mongodb.com/database-tools/mongoexport/>.
+
+- Export documents in JSON format to stdout:
+
+`mongoexport --uri={{mongodb_uri}} --collection={{collection_name}}`
+
+- Export documents that match with a query to a JSON file:
+
+`mongoexport --db={{database_name}} --collection={{collection_name}} --query="{{query_object}}" --out={{path/to/file.json}}`
+
+- Export documents to a JSON array instead of one object per line:
+
+`mongoexport --collection={{collection_name}} --jsonArray`
+
+- Export documents to a CSV file:
+
+`mongoexport --collection={{collection_name}} --type={{csv}} --fields="{{field1,field2,...}}" --out={{path/to/file.csv}}`
+
+- Export documents that match the query in the specified file to a CSV file, omitting a list of field names on first line:
```

Comment:
Perhaps this will read better?
```suggestion
- Export documents that match the query in the specified file to a CSV file, omitting the list of field names on the first line:
```